assignment/project/internal/service/post/data.go

```diff
@@ -20,11 +24,21 @@ func preparePostResponse(post *model.Post) *ct.PostResponse {
 	if post.CreatedAt != nil {
 		data.CreatedAt = post.CreatedAt.Format(time.RFC3339)
 	}
-
 	if post.UpdatedAt != nil {
 		data.UpdatedAt = post.UpdatedAt.Format(time.RFC3339)
 	}
 
+	// Convert user
+	data.User = prepareProfileResponse(post.User)
+
+	// Convert tags
+	if post.Tags != nil {
```

Comment:
nếu a không nhầm thì ko cần phải check khác nil,
for range sẽ ko loop nếu như slice là nil